zsh/plugins.zsh

```diff
@@ -0,0 +1,30 @@
+# Plugin loading with existence checks
+
+# Zsh syntax highlighting
+source_if_exists $HOME/.zsh/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh
+if [[ -f $HOME/.zsh/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh ]]; then
+    ZSH_HIGHLIGHT_STYLES[comment]='fg=blue'
+fi
+
+# Zsh autosuggestions
+source_if_exists $HOME/.zsh/zsh-autosuggestions/zsh-autosuggestions.zsh
+
+# FZF integration
+if command_exists fzf; then
+    # Source fzf key bindings and completion
+    if [[ -f /usr/share/fzf/key-bindings.zsh ]]; then
+        source /usr/share/fzf/key-bindings.zsh
+    elif [[ -f /usr/local/opt/fzf/shell/key-bindings.zsh ]]; then
+        source /usr/local/opt/fzf/shell/key-bindings.zsh
+    elif [[ -f $HOME/.fzf/shell/key-bindings.zsh ]]; then
+        source $HOME/.fzf/shell/key-bindings.zsh
+    elif command_exists fzf; then
```

Comment:
Don't we already know we have fzf because we checked for this at the begining of this block?